app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/confirmation_of_submission.scala.html

```diff
@@ -50,7 +50,7 @@
                         <h1 class="govuk-panel__title">@title</h1>
                         <div class="govuk-panel__body">
                             @messages(s"$key.reimbursement-amount")<br>
-                            <strong>@{formatAmountOfMoneyWithPoundSign(justSubmittedClaim.completeClaim.totalClaim.toDouble)}</strong>
+                            <strong>@{formatAmountOfMoneyWithPoundSign(200)}</strong>
```

Comment:
Not really sure what I'm looking at but should this be a hardcoded 200?

Reply:
will be fixed after the final journeys are in